src/Front-end/ListingsLayout.js

```diff
@@ -1,235 +1,264 @@
-import React, { useState, useMemo } from 'react';
-import { Typography, Container, Grid, Paper, TextField, Button, ThemeProvider, List, ListItem, Checkbox, ListItemText, Slider, Input, MenuItem, Select } from '@mui/material';
-import { CATEGORIES, CONDITIONS, SORTING, MAXPRICE } from './util';
```

Comment:
I think this is what's causing that error you posted on Slack! We should make sure we're using the stuff from util and the navigation bar from ./NavBar